### PR TITLE
[TE] Self-Serve tuning flow 03: preparing create-alert for new flow

### PR DESCRIPTION
**Pull request by newsummit**
For Create Alert, we will want to redirect to the Alert Page once its deployed. We will also want to change the way we trigger the alert function creation as well as replay (new API in progress). I added the new computed `alertFilterObj` to build the sensitivity settings for our new auto-tuning API into the filter object.

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs [hunk 1]
```diff
@@ -145,39 +144,88 @@
...
-        searchEnabled=false
-        selected=selectedSensitivity
-        onchange=(action (mut selectedSensitivity))
-        disabled=(not isMetricSelected)
-        required=true
-        as |sensitivity|
-      }}
+    {{#if isNewUx}}
+      <div class="te-form__vertical-block col-xs-12">
+        <label for="select-sensitivity" class="control-label te-label required">
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/template.hbs]: fix: `for` attribute doesn't match the input here

File: thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js [hunk 2]
```diff
@@ -7,10 +7,17 @@ import fetch from 'fetch';
 import Ember from 'ember';
 import moment from 'moment';
 import _ from 'lodash';
-import { checkStatus, buildDateEod } from 'thirdeye-frontend/helpers/utils';
+import Controller from '@ember/controller';
```
[the remark this answers is not in the document]
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js]: thanks. For some reason, I was blind to that :)

File: thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js [hunk 3]
```diff
@@ -7,10 +7,17 @@ import fetch from 'fetch';
...
+import { computed } from '@ember/object';
```
[the remark this answers is not in the document]
Comment (newsummit) [reply thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js]: I will need to address this in the next iteration.

**Comment by justYves** [comment]
Latest changes LGTM

**Pull request merged**